### DISCUSSION TITLE
client_x509_auth for ODoH

### DISCUSSION THREAD

**Discussion by dhruvp141188** [DNSCrypt/dnscrypt-proxy#1789] [Potential issues]
When trying to connect to ODoH Servers, they are required to have client cert verification. And right now there is no way to specify that in toml file.

**Comment by jedisct1** [comment]
> they are required

None of the relays (I guess you are referring to relays, not servers) require this as far as I know. That would be a little bit surprising.

Looking at the code, it seems like DoH queries and queries sent to ODoH relays use exactly the same code to connect, so credentials defined in the `[doh_client_x509_auth]` section should work for ODoH.
